### PR TITLE
Feat/20 our values section of website

### PR DESCRIPTION
**Pull request by jiayuu-u**
### Descriptions

- Link to UI design (if possible):

### Provide screenshots or videos that the function is working.

image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Comment by tatoMa** [comment]
The headers need to be capitalized, the same as the UI design.

The font-weight of the text "Our Values" seems not the same as our UI design.
image

**Review by tatoMa — COMMENTED** [review]
components/OurValue/OurValues.tsx is not a proper path for a component file.
Try components/OurValue/index.tsx and see how it goes.

File: styles/ListItem.module.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.item_list___without_bottom_border {
+  display: flex;
+  flex-direction: row;
+}
```
Comment (tatoMa) [inline styles/ListItem.module.css:4]: This is the best place to use the Pseudo-classes to define only the last item does not have a bottom border.

File: components/ItemList/ListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { textFont, titleFont } from "@/pages/_app";
+import styles from "@/styles/ListItem.module.css";
+import Image from "next/image";
+
+interface Props {
+  title: string;
+  content: string;
+  icon: string;
+  ifHasBottomBorder: boolean;
```
Comment (tatoMa) [inline components/ItemList/ListItem.tsx:9]: I don't think our data should all have this property.

**Comment by Cameron933** [comment]
LGTM. One more thing, now we decide to add the "components/*.tsx" path into jest.config.js file, this would let the jest run testing files under components folder. Therefore, may be including the whole section in one component test file is the better way which keeps our project consistent.

**Review by Cameron933 — APPROVED** [review]
LGTM

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**